src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTypeTableViewCell.swift

```diff
@@ -42,16 +42,23 @@ class DynamicTypeTableViewCell: UITableViewCell, DynamicTableViewTextCell {
 	func configureDynamicType(size: CGFloat = 17, weight: UIFont.Weight = .regular, style: UIFont.TextStyle = .body) {
 		contentTextLabel.font = UIFont.preferredFont(forTextStyle: style).scaledFont(size: size, weight: weight)
 		contentTextLabel.adjustsFontForContentSizeCategory = true
-		contentTextLabel.numberOfLines = 0
+		contentTextLabel.lineBreakMode = .byTruncatingTail
+		contentTextLabel.numberOfLines = 4
```

Comment:
That's not the best place to change the properties, because `DynamicTypeTableViewCell.swift` is a common `class.` 
You want a specific behaviour ;) 
So, if the changes solves your problem, maybe the changes aren't fit to other `UITableView`s that use that `class` behaviour.

Reply:
fixed.